src/widgets/helper/ChannelView.cpp

```diff
@@ -2578,7 +2578,11 @@ void ChannelView::addMessageContextMenuItems(QMenu *menu,
     if (!layout->getMessage()->id.isEmpty() && twitchChannel &&
         twitchChannel->hasModRights())
     {
-        menu->addAction(
+        menu->addSeparator();
+        auto *moderateAction = menu->addAction("Mo&derate");
```

Comment:
Hmm, we call it "Moderation" in the settings.

Reply:
My reasoning for writing "Moderate" was to fit with the rest of the right-click options which are written in action form

I'm open to:
a) Leave as is
b) Change to "Moderate..."
c) Change to "Moderation"
d) Change to "Moderation..."